ui/webui/test/ks.cfg

```diff
@@ -1,3 +1,4 @@
+liveimg --url="http://10.0.2.2:8000/live.tar"
```

Comment:
Btw. the tarball could be local if it would make the Web UI testing easier. Then it would be, for example:
`liveimg --url="file:///tmp/live.tar.xz"`

Reply:
This will however expect the file to be inside the test VM right?

Reply:
Yes.